src/main/resources/story/rest_api/API_Trello_Scenarios.story

```diff
@@ -0,0 +1,30 @@
+Description: Test demoing VIVIDUS capabilities for REST API
+
+
+GivenStories:/story/precondition/login_API.story#{id:login_api}
+
+
+Scenario: Verify creating the board
+When I execute HTTP POST request for resource with URL `https://api.trello.com/1/boards/?name=#{generate(numerify 'board##')}&key=${key}&token=${token}`
+When I save JSON element from `${response}` by JSON path `$.id` to story variable `board_id`
```

Comment:
Please, use consistent case for variable scope (either scope or SCOPE (as in precondition story) across all stories).